tool/inits/GEN_UM_IC_WITH_PHASE_WITH_TRANSFER_FUNC/README.txt

```diff
@@ -0,0 +1,13 @@
+The procedures of this LSS simulation includes:
+
+(a) Using CAMB/axionCAMB to produce transfer function (the input file planck_2018.ini/planck_2018_axion.ini for CAMB/axionCAMB is attached for reference)
```

Comment:
> How about providing the references or links for CAMB/axionCAMB in case some users may not know?
Good point! 
> And remind the users that we need to install them.
Added~